src/test/java/org/pesmypetcare/webservice/dao/petmanager/PetDaoTest.java

```diff
@@ -57,10 +61,40 @@
     private WriteBatch batch;
     @Mock
     private StorageDao storageDao;
+    @Mock
+    private DocumentSnapshot documentSnapshot;
 
     @InjectMocks
     private PetDao petDao = new PetDaoImpl();
 
+    @BeforeEach
+    public void setUp() {
+        petEntity = new PetEntity(GenderType.Female, "Huskie", "1996-01-08T12:20:30",
+            "One limb lost", "Constant attention", 85.0, null,
+            null);
+        petList = new ArrayList<>();
+        Map<String, Object> auxMap = new HashMap<>();
+        auxMap.put("name", PET_NAME);
+        auxMap.put("body", petEntity);
+        petList.add(auxMap);
+        petList.add(auxMap);
+        petList.add(auxMap);
+        snapshotList = new ArrayList<>();
+        snapshotList.add(documentSnapshot);
+        snapshotList.add(documentSnapshot);
+        snapshotList.add(documentSnapshot);
+        mealMap = new HashMap<>();
+        mealMap.put("mealName", "Asparagus");
+        mealMap.put("kcal", 84.0);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['84.0' is a magic number.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5517496)